frontend/components/PeerPrepNavbar.tsx

```diff
@@ -0,0 +1,56 @@
+import UserMenu from "@/components/UserMenu";
+import useUserInfo from "@/hooks/useUserInfo";
```

Comment:
is it okay if we store each component in a separate directory? e.g., `PeerPrepNavbar.tsx` can be stored under
`frontend/components/PeerPrepNavbar/index.tsx` instead:
```
├── frontend
│   ├── components
│   │   ├── PeerPrepNavbar
│   │   │   ├── index.tsx      # main code here
│   │   ├── UserMenu
│   │   │   ├── index.tsx      # main code here
```

this way the code base will be neater once we have components which are more complex and have subcomponents e.g.:
```
├── frontend
│   ├── components
│   │   ├── QuestionsTable
│   │   │   ├── DifficultyTab.tsx  
│   │   │   ├── QuestionCard.tsx  
│   │   │   ├── Selectors.tsx  
│   │   │   ├── index.tsx      # main code here
...
```

Reply:
sounds good! 👍🏻